src/containers/resources/ResourcesPage.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * @flow
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import { Colors, Typography } from 'lattice-ui-kit';
+
+import { RESOURCES, SUBTITLES } from '../../utils/constants/ResourcesConstants';
+import { trackLinkClick } from '../../utils/AnalyticsUtils';
+import { STATE } from '../../utils/constants/StateConstants';
+import { LABELS } from '../../utils/constants/Labels';
+import { getRenderTextFn } from '../../utils/AppUtils';
+import { ContentOuterWrapper, ContentWrapper, TextLink } from '../../components/layout';
+
+const { NEUTRAL } = Colors;
+
+const {
+  CALIFORNIA_RESOURCE_AND_REFERRAL_NETWORK,
+  CALIFORNIA_COVID19_CHILDCARE_RESPONSE,
+  CALIFORNIA_PARENT_AND_FAMILY_RESOURCES,
+  FAMILY_CHILD_CARE_HOME_LICENSING_INFORMATION,
+} = SUBTITLES;
+
+const Wrapper = styled(ContentWrapper)`
+  padding: 30px !important;
+  background-color: white;
+
+  h6 {
+    margin: 20px 0 5px;
+  }
+`;
+
+const Description = styled(Typography)`
+  color: ${NEUTRAL.N500};
+`;
```

Comment:
No change required here. `Typography` takes a `color` prop, but it's limited to like "textPrimary", "textSecondary", which doesn't quite work for us at this point.

Instead of creating a local component like this every time you need a custom color, I recommend moving this into `src/components/typography` and accept a `color` prop.

Reply:
isn't N500 textSecondary?